src/services/bundleAnalysis/useBundleAssets.tsx

```diff
@@ -266,32 +271,34 @@ export const useBundleAssets = ({
           dateBefore,
           dateAfter,
           filters,
-          assetsAfter: pageParam,
+          assetsAfter,
```

Comment:
Does this mean that `assetsAfter=null` doesn't get along well with the query? I'd think it might be worthwhile updating this API side if not. Would rather do that than do the few lines above on every infinite query.

Reply:
Sorry, `null` works fine, and I'm updating things rn to be set to `null` instead of `undefined`. The type for `pageParam` is inferred through the `initialPageParam` so we have to assign it to a `string` for the remaining cursors to keep the types happy.

Reply:
If null is fine, doesn't the old code work? `assetsAfter: pageParam`

Reply:
I may be misunderstanding though

Reply:
Soooo just did some testing, and the API is okay if just an empty string is provided ... so I'll get rid of all of this.

Reply:
NICE